pages/components/Typography/typography.css

```diff
@@ -0,0 +1,48 @@
+@import url(/css/main.css);
+
+.typography-preview-container {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    justify-content: space-evenly;
+    gap: 1rem;
+}
+
+/* Paragraph Text Styling */
+.para-lg{
+    font-size: 1.5rem;
+}
+.para-rg{
```

Comment:
In the code snippet, you have used para-xl para-lg but in the CSS file you have used para-lg para-rg. Just check it once